CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.1.3
+  - Improved performance when cloning events with big arrays
```

Comment:
I suggest "Improved performance by no longer cloning the source field if it is the same as the target field"